src/main/java/com/ldtteam/structurize/blocks/types/TimberFrameCentreType.java

```diff
@@ -41,8 +41,8 @@
     RED_TERRACOTTA(Blocks.RED_TERRACOTTA),
     BLACK_TERRACOTTA(Blocks.BLACK_TERRACOTTA),
     // Bricks
-    BRICK(Blocks.BRICKS, "Bricks"),
-    STONE_BRICK(Blocks.STONE_BRICKS, "Stone Bricks"),
+    BRICK(Blocks.BRICKS, "Brick"),
+    STONE_BRICK(Blocks.STONE_BRICKS, "Stone Brick"),
```

Comment:
The vanilla blocks are called "Bricks" and "Stone Bricks" tho :P

Reply:
I know. But there's, for example, `Stone Brick Slab` instead of `Stone Bricks Slab`.

Reply:
True, i guess it makes sense then :)